### PR TITLE
Update tutorial.md (render draw gameover)

### PR DESCRIPTION
**Pull request by EmrysMyrddin**
#### Checklist

* [x] Use a separate branch in your local repo (not `master`).
* [x] Test coverage is 100% (or you have a story for why it's ok).

### CODE REVIEW

File: docs/tutorial.md [hunk 1]
```diff
@@ -173,7 +173,7 @@ class TicTacToeBoard extends React.Component {
     let winner = '';
     if (this.props.ctx.gameover) {
       winner =
-        this.props.ctx.gameover.winner !== undefined ? (
+        this.props.ctx.gameover.draw  ? (
```
Comment (nicolodavis) [inline docs/tutorial.md]: Isn't this going to say "Draw!" when `this.props.ctx.gameover.draw` is `false`?

Comment (EmrysMyrddin) [reply docs/tutorial.md]: Yes you're totally right. I have fixed it.

**Comment by EmrysMyrddin** [comment]
No it's not really a bug.
It's just that the tutorial is showing how to return an object in the function `endGameIf`.

If the game is draw, it set a flag, but this flag is then not used. I found this a little confusing.

**Comment by nicolodavis** [comment]
In the meantime, I think it's ok to leave the code as is. The two states can either be Winner or Draw, so it's ok to just check:

```
if (winner) { display Winner }
else { display Draw }
```

rather than checking for the `draw` field explicitly.

**Comment by nicolodavis** [comment]
Feel free to continue the discussion here of course if you think we should change something.

**Pull request closed without merging**